src/routes/safe/store/actions/fetchSafe.js

```diff
@@ -59,16 +59,21 @@ export const checkAndUpdateSafe = (safeAddress: string) => async (dispatch: Redu
   localSafe.threshold = threshold.toNumber()
 
   dispatch(updateSafeThreshold({ safeAddress, threshold: threshold.toNumber() }))
+
   // If the remote owners does not contain a local address, we remove that local owner
   localOwners.forEach((localAddress) => {
-    if (!remoteOwners.includes(localAddress)) {
+    const remoteLowerCased = remoteOwners.map((owner) => owner.toLowerCase && owner.toLowerCase())
```

Comment:
You can use the `sameAddress` method from `src/logic/wallets/ethAddresses.js`

Reply:
Like here: #365

Reply:
indeed, thanks!